dotnet/src/Connectors/Connectors.AI.Oobabooga/TextCompletion/TextCompletionRequest.cs

```diff
@@ -173,5 +173,5 @@ public sealed class TextCompletionRequest
     /// In addition to the defaults. Written between "" and separated by commas. For instance: "\nYour Assistant:", "\nThe assistant:"
     /// </summary>
     [JsonPropertyName("stopping_strings")]
-    public List<string> StoppingStrings { get; set; } = new List<string>();
+    public IList<string> StoppingStrings { get; set; } = new List<string>();
```

Comment:
Do we do any conversion from StopSequences?

Reply:
`ICollection` would be my take :)

Reply:
No, we wouldn't do any conversion. If you want to use Oobabooga you would pass a `stopping_strings` property and if you are using OpenAI you would pass a `stop_sequences` property.

Reply:
Hmm that concerns me. I thought one of the few remaining things creating a SemanticFunction would yield me is the ability to use it between text completion models with little-to-no additional work required.